editor/js/libs/ui.three.js

```diff
@@ -155,14 +184,14 @@ class UITexture extends UISpan {
 				canvas.title = texture.sourceFile;
 				const scale = canvas.width / image.width;
 
-				if ( image.data === undefined ) {
+				if ( image.data !== undefined || texture.isCompressedTexture ) {
```

Comment:
I think we can write this line like so:
```js
if ( image.isDataTexture || texture.isCompressedTexture ) {
```
The test for `image.data` was added to ensure data textures from `RGBELoader` and `TGALoader` are supported.